didcomm/protocols/forward/forward.py

```diff
@@ -71,40 +269,47 @@ async def unpack_forward(
 
     :return: Forward plaintext
     """
```

Comment:
Please remove `UnsatisfiedConstraintError` from the doc. We don't use that exception anymore.

Reply:
done